contrib/vagrant/Makefile

```diff
@@ -0,0 +1,516 @@
+	     $(call DOWNLOAD_PKG_ARCHIVE,$(1)); \
+	     $(call CHECK_PKG_ARCHIVE,$(1)) \
+	        || { echo 'Wrong checksum!'; exit 1; }; \
+	else \
+		echo "ok"; \
+	fi
+
+endef
+$(foreach PKG,$(PKGS),$(eval $(call PKG_RULE,$(PKG))))
+
+# ===----------------------------------------------------------------------===
+
+GITIAN_HOST_FILES    := contrib/vagrant/Makefile \
+                        contrib/vagrant/gitian-host.json \
+                        contrib/vagrant/config/preseed.cfg \
+                        contrib/vagrant/config/vagrant.sh \
+                        contrib/vagrant/config/sshd.sh \
+                        contrib/vagrant/config/packages.sh \
+                        contrib/vagrant/config/cleanup.sh \
+                        contrib/vagrant/Vagrantfile
+GITIAN_HOST_FILES    := $(sort $(GITIAN_HOST_FILES))
+GITIAN_HOST_MANIFEST := $(shell $(call BUILD_MANIFEST,$(GITIAN_HOST_FILES)))
+
+_ := $(shell $(call \
+	VALIDATE_MANIFEST, \
+	.stamp-gitian-host.manifest, \
+	$(GITIAN_HOST_MANIFEST) \
+))
+
+.PHONY: gitian-host
+gitian-host: .stamp-gitian-host.manifest
+.stamp-gitian-host.manifest: \
+	.stamp-check-requirements.h \
+	$(foreach FILE,$(GITIAN_HOST_FILES),$(shell echo $(FILE) | $(SED) 's:^contrib/vagrant/::g'))
+	
+	-$(call VAGRANT_DESTROY)
+	
+	# Remove any temporary files from interupted builds that would
+	# interfere if present.
+	if [ -e output-gitian-host-xenial64 ]; then \
+	    rm -rf output-gitian-host-xenial64; \
+	fi
+	
+	packer build gitian-host.json
+	vagrant box add --name gitian-host-xenial64 --force box/virtualbox/gitian-host-xenial64-0.box
+	-rm -rf box
+	
+	$(call VAGRANT_UP)
+	
+	-$(call VAGRANT_HALT)
+	
+	echo $(GITIAN_HOST_MANIFEST) > "$@"
+
+# ===----------------------------------------------------------------------===
+
+LXC_BASE_FILES    := contrib/vagrant/Makefile
+LXC_BASE_FILES    := $(sort $(LXC_BASE_FILES))
+LXC_BASE_MANIFEST := $(shell $(call BUILD_MANIFEST,$(LXC_BASE_FILES)))
+
+_ := $(shell $(call \
+	VALIDATE_MANIFEST, \
+	.stamp-lxc-base.manifest, \
+	$(LXC_BASE_MANIFEST) \
+))
+
+.PHONY: lxc-base
+lxc-base: .stamp-lxc-base.manifest
+.stamp-lxc-base.manifest: \
+	.stamp-gitian-host.manifest \
+	$(foreach FILE,$(LXC_BASE_FILES),$(shell echo $(FILE) | $(SED) 's:^contrib/vagrant/::g'))
+	
+	$(call VAGRANT_UP)
+	
+	vagrant ssh --command "\
+	    if [ -d ~/gitian-builder ]; then \
+	        rm -rf ~/gitian-builder; \
+	    fi"
+	vagrant ssh --command "\
+	    git clone 'https://github.com/devrandom/gitian-builder' ~/gitian-builder"
+	vagrant ssh --command "cd ~/gitian-builder && \
+	    git checkout e2172de8"
+	vagrant ssh --command "cd ~/gitian-builder && \
+	    ln -s $(GUEST_CACHE_DIR) inputs"
+	
+	vagrant ssh --command "\
+	    cd ~/gitian-builder; \
+	    if [ ! -f $(GUEST_CACHE_DIR)/lxc-base/$(LXC_BASE_MANIFEST)/base-vm.tar.gz ]; then \
+	        env LXC_BRIDGE=lxcbr0 LXC_GUEST_IP=10.0.3.2 _HOST_IP=10.0.3.1 \
+	            bin/make-base-vm --lxc --suite trusty --arch amd64 || exit 1; \
```

Comment:
Should this say bionic?

Reply:
Sure, but on master the gitian descriptors use bionic instead of trusty, which is why I left the note.

Reply:
Ah, I see. Should I close this PR but leave the fork available, for now, for others to work off of?